core/account-id/src/lib.rs

```diff
@@ -401,7 +452,7 @@ impl<'a> arbitrary::Arbitrary<'a> for AccountId {
 mod tests {
     use super::*;
 
-    pub const OK_ACCOUNT_IDS: [&str; 24] = [
+    pub const OK_ACCOUNT_IDS: [&str; 25] = [
```

Comment:
nit: Would _ work in place of the explicit number of items in the array?

Reply:
It compiles, but
```
using `_` for array lengths is unstable
see issue #85077 <https://github.com/rust-lang/rust/issues/85077> for more information
lib.rs(459, 38): consider specifying the array length: `25`
```